File: app/src/main/java/edu/uw/layoutdemo/MainActivity.java
package edu.uw.layoutdemo;

import android.os.Bundle;
import android.support.v7.app.AppCompatActivity;
import android.view.View;
import android.view.ViewStub;
import android.widget.Button;

public class MainActivity extends AppCompatActivity {

    public static final String TAG = "MainActivity";

    @Override
    protected void onCreate(Bundle savedInstanceState) {
        super.onCreate(savedInstanceState);
        setContentView(R.layout.activity_main);
//        setContentView(R.layout.activity_relative);
//        setContentView(R.layout.input_control_layout);

        Button btn = (Button)findViewById(R.id.button);
        btn.setOnClickListener(new View.OnClickListener() {
            @Override
            public void onClick(View v) {
                ViewStub stub = (ViewStub) findViewById(R.id.stub);
                if(stub != null) {
                    View inflated = stub.inflate();
                }
            }
        });
    }
}

File: app/src/main/java/edu/uw/layoutdemo/MainActivity.kt
package edu.uw.layoutdemo

import android.os.Bundle
import android.support.v7.app.AppCompatActivity
import android.util.Log
import android.widget.Button

class MainActivity : AppCompatActivity() {

    private val TAG = "MainActivity"

    override fun onCreate(savedInstanceState: Bundle?) {
        super.onCreate(savedInstanceState)
        setContentView(R.layout.activity_main)

        val myB = findViewById<Button>(R.id.button)
        Log.v(TAG, myB.text.toString())
    }
}
